### PR TITLE
Adding my first hw to dev

### PR DESCRIPTION
**Pull request by 28daniyar**
My first HW

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+let currentTimeDoc = document.getElementById('currentTime');
+let currentDateDOc = document.getElementById('currentDate');
+
+const weekday = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+const monthNames = ["January", "February", "March", "April", "May", "June",
+    "July", "August", "September", "October", "November", "December"];
+
+today = new Date();
+let dayOfWeek = weekday[today.getDay()];
```
Comment (dkrupenya-quantori) [inline assets/js/script.js:9]: 👍

File: assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+let currentTimeDoc = document.getElementById('currentTime');
```
Comment (dkrupenya-quantori) [inline assets/js/script.js]: Use `const` by default.
You are not going to change it.

**Review by dkrupenya-quantori — APPROVED** [review]

**Comment by dkrupenya-quantori** [comment]
nothing to comment. Good solid work

File: assets/js/script.js (hunk 2 above)
Comment (28daniyar) [reply assets/js/script.js]: I created new PR with this changings, thank you! Should i merge this new PR and create new branch for new task?

**Pull request closed without merging**